public_dropin_environments/r_lang/requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy>=1.20.3
+numpy>=1.20.3,<2.0  # The upper bound is set due to pandas constraint
```

Comment:
did you intend to keep the numpy 2.0 upper constraint for `r_lang` dropin env requirements?